### DISCUSSION TITLE
Error sending confirmation email: gomail: could not send email 1: 450 Unexpected token

### DISCUSSION THREAD

**Discussion by jonathanbouwens** [supabase/supabase#28554] [Questions] [answered]
Hi!

I have Supabase linked with Loops and suddenly it can't send the magic link e-mail anymore. I have not changed the payload, transctionId, confirmationUrl, etc in the email template. 

When I disable SMPT, I receive the confirmation e-mail from Supabase so I assume something is going wrong between Supabase and Loops. 

Does anybody know the solution to this?

auth_event":{"action":"user_confirmation_requested","actor_id":"43cccde7-15eb-4e5c-ac0d-8b6c31c89a48","actor_username":"j****@***.com","actor_via_sso":false,"log_type":"user","traits":{"provider":"email"}},"component":"api","error":"Error sending confirmation email: gomail: could not send email 1: 450 Unexpected token j in JSON at position 59","level":"error","method":"POST","msg":"500: Error sending confirmation mail","path":"/otp","referer":"https://repurpost.io/auth/confirm?next=/home","remote_addr":"44.200.112.112","request_id":"8b20751a038fc957-IAD","time":"2024-08-12T12:27:06Z"}

**Comment by GaryAustin1** [comment]
I don't know anything about Loops.
This guide implies they have a special format for emails they need.  This may be to only use there transactional emails, I did not search further.
https://loops.so/docs/integrations/supabase
If you are using those then you have to have special templates with info in them.

**Reply by jonathanbouwens** [reply]
That is exactly what I have done and it was also working. 

<img width="856" alt="Screenshot 2024-08-12 at 15 08 21" src="https://github.com/user-attachments/assets/0edb9da8-5a14-4f61-8900-c351895aa1e1">

**Reply by GaryAustin1** [reply, marked as answer]
Their example shows quotes around the {{...}} also....

![image](https://github.com/user-attachments/assets/87deab34-7c26-45f5-b905-8381646693c4)

But if it worked for you before not sure how that would impact things.

**Reply by jonathanbouwens** [reply]
Thats it! Man I feel stupid haha

**Reply by jonathanbouwens** [reply]
Thanks!
